aviary/variable_info/variable_meta_data.py

```diff
@@ -5775,11 +5775,12 @@
 )
 
 add_meta_data(
-    Aircraft.Wing.MOUNTING_TYPE,
+    Aircraft.Wing.MOUNT_LOCATION,
     meta_data=_MetaData,
     historical_name={"GASP": 'INGASP.HWING', "FLOPS": None, "LEAPS1": None},
     units='unitless',
-    desc='wing location on fuselage (0 = low wing, 1 = high wing)',
+    desc='wing mount location on fuselage (0 = low wing, 1 = high wing)',
```

Comment:
This is a continuous variable. in the description we should add:
"Values between 0 and 1 are acceptable".